Angular/src/app/shared/api-endpoint.constants.ts

```diff
@@ -7,3 +6,4 @@ export const requestUrl = baseUrl + "request";
 export const categoryUrl = baseUrl + "category";
 export const participUrl = baseUrl + "participation";
 export const paymentUrl = baseUrl + "payment";
+export const typeProjectUrl = `${baseUrl}typeProject/`;
```

Comment:
please, keep consistency or change everything else:
```suggestion
export const typeProjectUrl =baseUrl + "typeProject/";
```